Purchases/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -22,20 +22,34 @@ public typealias SK1Product = SKProduct
 @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
```

Comment:
I don't have a strong opinion on this, but this file contains a lot of definitions and could be difficult to maintain. Would it make sense to split this file into small files, at least `StoreProduct` and `StoreProductType`?

WDYT? 🤔

Reply:
That's a great point, I think it makes sense to take out the subclasses to another file 👍

Reply:
This is the only remaining TODO

Reply:
Done!